packages/cli/src/mcp-config.ts

```diff
@@ -0,0 +1,113 @@
+import { readJSON } from "fs-extra"
```

Comment:
use genaiscriptDebug and add debug statements in the file to trace the mcp server resolution